.github/workflows/ci_workflows.yml

```diff
@@ -9,45 +9,27 @@ on:
 
 jobs:
   tests:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: ubuntu-latest
-            python-version: 3.7
-            toxenv: py37-test-pytest46
-          - os: windows-latest
-            python-version: 3.7
-            toxenv: py37-test-pytest50
-          - os: macos-latest
-            python-version: 3.8
-            toxenv: py38-test-pytest52
-          - os: ubuntu-latest
-            python-version: 3.8
-            toxenv: py38-test-pytest53
-          - os: windows-latest
-            python-version: 3.9
-            toxenv: py39-test-pytest60
-          - os: macos-latest
-            python-version: 3.9
-            toxenv: py39-test-pytest61
-          - os: ubuntu-latest
-            python-version: '3.10'
-            toxenv: py310-test-pytest62
-          - os: ubuntu-latest
-            python-version: '3.10'
-            toxenv: py310-test-pytestdev
-
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install tox
-      run: python -m pip install tox
-    - name: Run tox
-      run: tox -v -e ${{ matrix.toxenv }}
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
+    with:
+      envs: |
+        - linux: py37-test-pytest46
```

Comment:
Somewhat unrelated improvement: instead of `pytest46` I like to call it `pytestoldest`, and pin it to the actual stated minimum version rather than a wildcard in that series. E.g. ``pytest==4.6.0``

If a fix is needed from an actual bugfix, then bump the minimum version to reflect that.

(for the rest of the versions I feel the flexible wildcard is fine, we need a wide enough matrix to cover over versions but don't need to test everything).

Reply:
Sounds good, will do